### PR TITLE
Update checklists

### PR DESCRIPTION
**Pull request by rrhyne**
Add visual design checklist, reference NN's 10 usability heuristics

### CODE REVIEW

File: handbook/product/design/design-process.md [hunk 1]
```diff
@@ -100,12 +101,19 @@ While asynchronous communication is a core attribute of remote work, key moments
...
   - Process
       - A GitHub issue should be created to track the work
-      - Wireframes are produced in Figma
-      - Wireframes will be announced in Slack and linked in the GitHub issue well before they are complete for review
-      - Schedule and conduct a research presentation meeting
+      - Designs are produced in Figma
+      - Designs will be announced in Slack and linked in the GitHub issue well before they are complete for review
+      - Ensure designs meet the visual design checklist:
+        - Ensure spacing is consistent and matches the 8pt grid system
+        - Ensure colors match existing styles
```
Comment (AlicjaSuska) [inline handbook/product/design/design-process.md:108]: How about mentioning the text styles here as well?

Ensure text styles match the existing ones, if possible. Introduce new text styles only if really necessary. Remember to add them to the design system with proper description and ensure that they meet the accessibility standards.

File: handbook/product/design/design-process.md [hunk 2]
```diff
@@ -100,12 +101,19 @@ While asynchronous communication is a core attribute of remote work, key moments
...
+          - If new styles are introduced, label them clearly
+        - Review designs for accessability:
+          - Color blind check
+          - Contrast check
```
Comment (AlicjaSuska) [inline handbook/product/design/design-process.md:112]: I would specify here: 'Aim to meet at least AA standard for small text'

File: handbook/product/design/design-process.md [hunk 3]
```diff
@@ -100,12 +101,19 @@ While asynchronous communication is a core attribute of remote work, key moments
...
-      - Wireframes will be announced in Slack and linked in the GitHub issue well before they are complete for review
-      - Schedule and conduct a research presentation meeting
+      - Designs are produced in Figma
+      - Designs will be announced in Slack and linked in the GitHub issue well before they are complete for review
+      - Ensure designs meet the visual design checklist:
+        - Ensure spacing is consistent and matches the 8pt grid system
+        - Ensure colors match existing styles
+          - If new styles are introduced, label them clearly
+        - Review designs for accessability:
+          - Color blind check
```
Comment (AlicjaSuska) [inline handbook/product/design/design-process.md:111]: How would you like it to be checked?

Comment (rrhyne) [reply handbook/product/design/design-process.md:112]: Done! Please feel free to make non-controversial changes like this without commenting!

File: handbook/product/design/design-process.md (hunk 1 above)
Comment (rrhyne) [reply handbook/product/design/design-process.md]: Great addition, I generalized this comment to refer to all styles and added accessibility standards note.

File: handbook/product/design/design-process.md (hunk 2 above)
Comment (rrhyne) [reply handbook/product/design/design-process.md]: Done!

**Pull request merged**